BurnOutSharp/ProtectionType/SafeDisc.cs

```diff
@@ -221,39 +774,67 @@ public static string GetSecdrvVersion(string firstMatchedString, IEnumerable<str
             FileInfo fi = new FileInfo(firstMatchedString);
             switch (fi.Length)
             {
+                // Found in Redump entries 9718, 12885, 21154, 31149, 37523, 37920.
+                case 14_304:
+                    return "/ SafeDisc 1.06.000-1.20.001";
```

Comment:
Did you mean to have the leading `/` on all of these? What does the typical output from each of these look like with that?

Reply:
It is intentional, and outputs as “SafeDisc Security Driver X.YY.ZZZ / SafeDisc A.BB.CCC”, or simply as “SafeDisc Security Driver / SafeDisc A.BB.CCC” if there is no discernible version for the driver.

Reply:
Ah okay, so the place that it's being used from `SafeDisc Security Driver`. Okay, that's fine then.